### ISSUE TITLE
Multi-monitor: setStrut() doesn't work if the position is Right

### ISSUE THREAD

**Issue by dangvd** [dangvd/ksmoothdock#24]
It sets the strut on the right monitor instead of the left monitor (that the dock is on).

**Comment by dangvd** [comment]
Seems tricky to fix, as KWindowSystem::setStrut() seems to only work on the whole combined screen space.

**Comment by mihanich** [comment]
Faced with same issue when dock is on left side. Set autohide - and can not get dock back.

**Comment by dangvd** [comment]
Yeah if the dock is on the left side of the 2nd screen, it doesn't work either for the same reason.
